packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -405,6 +456,12 @@ export class Dropdown implements ComponentInterface, DropdownInterface {
     );
   }
 
+  private focusDropdownItem(index: number) {
+    requestAnimationFrame(() => {
+      this.dropdownItems[index]?.shadowRoot.querySelector('button').focus();
```

Comment:
DropdownItem's implements DropdownItemWrapper interface which gives you access to the `getDropdownItemElement` maybe a better solution as query the shadowRoot?

Reply:
Need to focus the nested button for the focus-visible state to work